### DISCUSSION TITLE
I can't use any other prompt besides the default one

### DISCUSSION THREAD

**Discussion by Samthesurf** [JanDeDobbeleer/oh-my-posh#4588] [Troubleshoot]
I wrote my zsh prompt as follows
eval "$(oh-my-posh init zsh --config '~/clean-detailed.omp.json')" but all it shows me when I source my zshrc is config error, it worked well on my Linux Mint install but when I installed ArchLinux, it no longer worked and it only showed me config error,
Operating system: Arch Linux x86_64
Kernel: 6.1.70-1-lts
Terminal: Wezterm
Shell: Zsh

**Comment by JanDeDobbeleer** [comment]
@Samthesurf you can check why it says so using `oh-my-posh debug --config '~/clean-detailed.omp.json'`

**Reply by Samthesurf** [reply]
Why I checked the logs I saw an error it says  config.go:loadConfig:160 - open /home/samuelsurf/clean-detailed.omp.json.omp.json: permission denied

**Reply by JanDeDobbeleer** [reply]
Why is the extension added twice? Either way, that file has permissions restricting access to the executing user.
